process/transactions/logsAndEventsProcessorNFTs.go

```diff
@@ -0,0 +1,130 @@
+package transactions
+
+import (
+	"encoding/hex"
+	"math/big"
+
+	"github.com/ElrondNetwork/elastic-indexer-go/data"
+	"github.com/ElrondNetwork/elastic-indexer-go/disabled"
+	"github.com/ElrondNetwork/elrond-go/core"
+	dataElrond "github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/marshal"
+	"github.com/ElrondNetwork/elrond-go/process"
+	"github.com/ElrondNetwork/elrond-go/sharding"
+)
+
+type logsAndEventsProcessor struct {
+	pubKeyConverter          core.PubkeyConverter
+	marshalizer              marshal.Marshalizer
+	logsAndEventsHandler     process.TransactionLogProcessorDatabase
+	nftOperationsIdentifiers map[string]struct{}
+	shardCoordinator         sharding.Coordinator
+}
+
+func newLogsAndEventsProcessorNFT(
+	shardCoordinator sharding.Coordinator,
+	pubKeyConverter core.PubkeyConverter,
+	marshalizer marshal.Marshalizer,
+) *logsAndEventsProcessor {
+	return &logsAndEventsProcessor{
+		shardCoordinator:     shardCoordinator,
+		pubKeyConverter:      pubKeyConverter,
+		marshalizer:          marshalizer,
+		logsAndEventsHandler: disabled.NewNilTxLogsProcessor(),
```

Comment:
Kind of misleading. We have the structure `logsAndEventsProcessor` that contains a `logsAndEventsHandler`

Reply:
renamed.